karma.conf.js

```diff
@@ -103,39 +54,30 @@ module.exports = function(config) {
 
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ['mocha', 'sinon', 'chai', 'sinon-chai'],
-
+    frameworks: ['requirejs', 'mocha', 'chai'],
```

Comment:
Shouldn't this be done together with an earlier commit? Otherwise I assume there are a few commits that fail the tests?